src/main/java/model/Comparator.java

```diff
@@ -0,0 +1,22 @@
+package main.java.model;
+
+/**
+ * Simple enumerator for comparators. Available comparators are GT (Greater
+ * Than), LT (Lesser Than), EQ (Equal) and DF (Different).
+ * 
+ * @author hugo.barroca
+ *
+ */
+public enum Comparator {
+	GT(">"), LT("<"), EQ("=="), DF("!=");

```

Comment:
não vamos ter GE e LE (greater or equal, lesser or equal)?

Reply:
also da-lhe outro nome q nao Comparator - Operator would be better

Reply:
Eu tinha colocado Operator originalmente, mas como eles não fazem nenhuma operação tirando a comparação fiquei na dúvida... Vou mudar para Operator though, também acho que fica melhor.